saos-import/src/main/java/pl/edu/icm/saos/importer/notapi/common/content/ContentDownloadStepExecutionListener.java

```diff
@@ -0,0 +1,96 @@
+package pl.edu.icm.saos.importer.notapi.common.content;
+
+import java.io.File;
+import java.util.Collection;
+
+import org.springframework.batch.core.ExitStatus;
+import org.springframework.batch.core.StepExecution;
+import org.springframework.batch.core.StepExecutionListener;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import pl.edu.icm.saos.importer.notapi.common.ImportFileUtils;
+import pl.edu.icm.saos.persistence.model.importer.RawSourceJudgment;
+
+import com.google.common.collect.Lists;
+
+/**
+ * Listener responsible for copying content files.
+ * 
+ * @author madryk
+ */
+public class ContentDownloadStepExecutionListener implements StepExecutionListener {
+    
+    private ImportFileUtils importFileUtils;
+    
+    private ImportContentFileUtils importContentFileUtils;
+    
+    private ContentSourceFileFinder importContentFileFinder;
+    
+    private String importDir;
+    
+    private String importContentDir;
```

Comment:
We should use consistent terminology for these the directories across the app. 
For example sourceImportMetadataDir, sourceImportContentDir, importDownloadContentDir